src/tsickle.ts

```diff
@@ -2153,26 +2153,39 @@ function addClutzAliases(
       // Want to alias the symbol to match what clutz would produce, so clutz .d.ts's
       // can reference symbols from typescript .d.ts's. See examples at:
       // https://github.com/angular/clutz/tree/master/src/test/java/com/google/javascript/clutz
-      const clutzSymbolName = 'module$contents$' + clutzModuleName + '_' + symbol.name;
+      // The first symbol name is that currently produced by clutz, and the second
+      // is what incremental clutz will produce.
+      const clutzSymbolNames = [
+        'module$contents$' + clutzModuleName + '_' + symbol.name,
+        'module$exports$' + clutzModuleName + '.' + symbol.name
+      ];
 
       const {params, paramsWithContraint} = getGenericTypeParameters(symbol);
 
       if (symbol.flags & ts.SymbolFlags.Class) {
         // classes need special care to match clutz, which seperates class types into a
         // type for the static properties and a type for the instance properties
-        reexports.push(`type ${clutzSymbolName}${paramsWithContraint} = ${symbol.name}${params};`);
-        reexports.push(`const ${clutzSymbolName}: typeof ${symbol.name};`);
-        reexports.push(
-            `type ${clutzSymbolName}_Instance${paramsWithContraint} = ${symbol.name}${params};`);
-        reexports.push(`const ${clutzSymbolName}_Instance: typeof ${symbol.name};`);
+        for (const clutzSymbolName of clutzSymbolNames) {
```

Comment:
these loops are a bit odd for N=2 (and never more). Consider (optional) extracting the 4 push statements here into a function declare in this scope that you call twice with the two symbols, and for the code below just duplicating the `reexports.push` line. Roughly the same amount of code, but arguably more clear than having a loop for two items.